src/StreamingBodyParser/ParserInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace React\Http\StreamingBodyParser;
+
+use Evenement\EventEmitterInterface;
+use React\Http\Request;
+
+interface ParserInterface extends EventEmitterInterface
+{
+    public function __construct(Request $request);
```

Comment:
Since the parser implementations are now marked as `@internal`, i propose to remove `create` from the interface since this is now just a additional redundant factory method.